src/ucp/core/ucp_ep.c

```diff
@@ -521,11 +536,13 @@ ucp_ep_create_api_conn_request(ucp_worker_h worker,
 out_ep_destroy:
     ucp_ep_destroy_internal(ep);
 out:
-    if (status == UCS_OK) {
-        status = uct_iface_accept(conn_request->uct_iface,
-                                  conn_request->uct_req);
-    } else {
-        uct_iface_reject(conn_request->uct_iface, conn_request->uct_req);
+    if (!ucp_worker_sockaddr_is_cm_proto(worker)) {
```

Comment:
if, for example, ucp_address_unpack() failed in ucp_ep_create_accept, how can we call uct_iface_accept?

Reply:
we can't, `uct_iface_reject` will be called. `ucp_ep_create_accept` in CM mode always returns `UCS_ERR_NOT_IMPLEMENTED` for now. Connection establishment and reject flow will be added in next PR.